Update learn-guides-assets-create.md (#5451)
w3f/polkadot-wiki: docs/learn/learn-guides-assets-create.md

:::info

Before minting assets on the Asset Hub, we recommend that you try out this tutorial on Westmint,

> which is a systems parachain on Westend. The WND tokens (Westies) used for Westmint are free and are available through a

Reviewer: ```suggestion
which is a system parachain on Westend test network. The WND tokens (Westies) can be teleported from Westend to Westmint, are available for free through a
```
